### ISSUE TITLE
Add a location descriptor attribute to gene descriptors

### ISSUE THREAD

**Issue by ahwagner** [ga4gh/vrsatile#18]
@larrybabb REALLY wants this

**Comment by sumit-158** [comment]
Hey, @ahwagner Is anyone is working on this! if not can I work on this?? Ans also can you help me to out how to start??

**Comment by ahwagner** [comment]
Please do! We use [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow) for development. This is a great issue for an introduction to adding to VRSATILE.

The Descriptor classes are all maintained in https://github.com/ga4gh/vrsatile/blob/main/schema/vod-source.yaml. We have not transferred over Gene and Location Descriptors from the documentation yet; follow the template in that YAML to add the YAML representation in the Gene and Location descriptors per their current information models [here](https://vrsatile.readthedocs.io/en/latest/value_object_descriptor/index.html#location-descriptor). Also, I would add both Gene Descriptor and Location Descriptor to the `oneOf` array [here](https://github.com/ga4gh/vrsatile/blob/main/schema/vod-source.yaml#L13) when making those changes.

**Comment by sumit-158** [comment]
Sorry! @ahwagner to bother you again but, I am confuse  what exactly I have to do?

**Comment by sumit-158** [comment]
@ahwagner  Did I have to change "vodjson" file also?

**Comment by sumit-158** [comment]
@ahwagner is there any problem in this pull request! why no one have review it yet?

**Comment by ahwagner** [comment]
> @ahwagner Did I have to change "vodjson" file also?

yes; please see the PR for instructions on building the YAML / JSON / .rst artifacts from source.
